urdf_model/include/urdf_model/utils.h

```diff
@@ -0,0 +1,67 @@
+/*********************************************************************
+* Software License Agreement (BSD License)
+*
+*  Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
The general rule is that, when making non-trivial changes to existing code, you add a copyright line with appropriate owner and year. Never remove a copyright line from existing code.

But if it's new code, just create a new copyright line with appropriate content (e.g., OSRF, 2016).